samples/connectedhomeip/lock/README.rst

```diff
@@ -0,0 +1,186 @@
+.. _connectedhomeip_lock_sample
+
+Connected Home over IP: Lock
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Connected Home over IP Lock sample demonstrates the usage of 
```

Comment:
One sentence per line, without breaking it, as @greg-fer said.

Reply:
Done